examples/navigation/src/epics/adminAccess.test.js

```diff
@@ -0,0 +1,26 @@
+import { push } from 'react-router-redux'
+import { TestScheduler } from 'rxjs/testing'
+import { accessDenied, CHECKED_ADMIN_ACCESS } from '../actions'
+import adminAccessEpic from './adminAccess'
+
+describe('adminAccess epic', () => {
+  it('creates an accessDenied action first, then a route change action', done => {
+    const testScheduler = new TestScheduler((actual, expected) => {
+      expect(actual).toEqual(expected)
+      done()
```

Comment:
Did you find his was necessary? It shouldn’t be.

Reply:
That is, TestScheduler usage should run entirely synchronously since time is virtualized by it.

Reply:
Oops, good spot, have removed it 👁